client/state/countries/reducer.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { createReducer, combineReducers } from 'state/utils';
+import {
+	COUNTRIES_DOMAINS_UPDATED,
+	COUNTRIES_PAYMENTS_UPDATED,
+	COUNTRIES_SMS_UPDATED,
```

Comment:
How about `_ADD` instead of `_UPDATE` here?

Reply:
or maybe `_UPDATE` ? I suppose we are blindly updating here.

Reply:
Yep. It's pretty much all or nothing. It's just a list rather than a process where we are accumulating countries.